crmsh/ui_context.py

```diff
@@ -255,6 +255,9 @@ def enter_level(self, level):
         self._in_transit = True
 
         entry = level()
+        if self.command_name == 'configure' and self.command_args and self.command_args[0] in ['delete']:
```

Comment:
I found issue https://github.com/ClusterLabs/crmsh/issues/1466 might have the same root cause
It works after running `configure refresh`

So here we should think a common solution, not only for the `delete` command

Reply:
Yes, https://github.com/ClusterLabs/crmsh/pull/1403 cause this and #1466 both

Reply:
Hi @liangxin1300 , could you please check the update?